variables.tf

```diff
@@ -125,3 +125,12 @@ variable "analytics_destination_type" {
   default     = "Dedicated"
   description = "Log analytics destination type"
 }
+
+variable "managed_private_endpoint" {
+  type = map(object({
```

Comment:
consider using set of objects instead of map of objects, for example:
```hcl
type = set(object({
  name = string
  target_resource_id = string
  subresource_name   = string
}))
```

Reply:
done